src/Cowsay.zig

```diff
@@ -142,30 +122,26 @@ pub fn useCowFile(self: *Self, filename: []const u8) void {
     const file = std.fs.cwd().openFile(filename, .{}) catch |err| {
         // use default
         self.useDefaultCow();
-        _ = err catch {};
+        std.log.err("File \"{s}\": {s}", .{ filename, @errorName(err) });
         return;
     };
     defer file.close();
     const n_read = file.readAll(&self.cow_buffer) catch |err| {
         // use default
         self.useDefaultCow();
-        _ = err catch {};
+        std.log.err("File \"{s}\": {s}", .{ filename, @errorName(err) });
         return;
     };
-    self.cow_buffer_length = n_read;
+    self.cow = self.cow_buffer[0..n_read];
 }
 /// Use the default cow
 pub fn useDefaultCow(self: *Self) void {
-    self.cow_buffer_length = default_cow.len;
-    for (default_cow, 0..) |c, i| {
-        self.cow_buffer[i] = c;
-    }
+    self.cow = default_cow;
```

Comment:
You can use a `@memcpy` instead of copying to buffer character by character. Even better, now that you use a `cow` slice rather than manually keeping track of buffer length, you can simply make the slice point to the default cow!